src/app/config/features/payment.config.ts

```diff
@@ -45,7 +45,7 @@ const paymentFeature: Schema<IStripe> = {
   maxPaymentAmountCents: {
     doc: 'Maximum amount that can be paid for a form',
     format: Number,
-    default: 100000, // $1000, due to IRAS limit for invoice
+    default: Number.MAX_VALUE,
```

Comment:
LOL, let's have a default a bit saner than that 😅 .

```javascript
Number.MAX_VALUE
1.7976931348623157e+308
```

I'll just suggest arbitrarily 1M; but realistically, I'd rather see 100k or even 10k, and getting notified if anyone is collecting more than 1k 😰

Reply:
Set it to 1M in [e5af962](https://github.com/opengovsg/FormSG/pull/6407/commits/e5af96257a3e27aad04365b00ce9bc774a11aba6), we can adjust this up/down by changing the env var anyway.